### DISCUSSION TITLE
Admin area suggestions

### DISCUSSION THREAD

**Discussion by Tigerlady** [MasaCMS/MasaCMS#130] [Ideas]
So I have been working with MASA for a while now with my partner, who does a lot of edits.  We have some suggestions concerning the admin edit page with the right-hand accordion.  

1. It would be nice to have the Summary area show the editor in the body area of the page instead of having the editor show up in the accordion. sort of like it does when looking at the Extended Attributes section.
2. I have the MASA Translation plugin, and when expanding the section, you can't see the entire section of pages that have been assigned.  Again it would be nice to have this show up in the main body area like the Extended Attributes.  ![image](https://user-images.githubusercontent.com/1850313/192812247-4ffed323-4e99-4c2b-9dcf-a486a786795a.png)

**Comment by guustnieuwenhuis** [comment]
1. Doesn't it help enough to make the sidebar wider?
2. We'll tackle this issue in a next version of the plugin.

**Comment by Tigerlady** [comment]
I didn't know the sidebar could be expanded until my SEO person found it by mistake.  Secondly, when you have pages that are several layers deep, you can't see the entire URL, even on expanded screens.  It does help that you can expand the sidebar but not quite enough for deep layers.  See screenshot.

![image](https://user-images.githubusercontent.com/1850313/198054395-3dab26b1-0d8b-4f2c-a9b7-d00cebfea96e.png)
